feedback/ngffeedback.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021 Jolla Ltd.
+ *
+ * This file is part of the libngf QtFeedback plugin.
+ * Based on qt5-feedback-hapticks-droid-vibrator.
+ *
+ * This work is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This work is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this work; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+ */
+
+#ifndef NGF_FEEDBACK_H
+#define NGF_FEEDBACK_H
+
+#include <QObject>
+#include <QLoggingCategory>
+#include <qfeedbackplugininterfaces.h>
+#include "ngfclient.h"
+
+Q_DECLARE_LOGGING_CATEGORY(ngflc)
```

Comment:
Doesn't matter too much but since it's only the single c++ file doing logging, I'd keep the logging details on that side until there's need elsewhere.

Reply:
Yeah, removed this line.